clients/python/test/test_osparc/test_basic.py

```diff
@@ -1,6 +1,56 @@
+import json
+import subprocess
+from pathlib import Path
+from typing import Any, Dict, List, Set
+
 import osparc
 
+_PYTHON_DIR: Path = Path(__file__).parent.parent.parent
+
 
 def test_get_api():
     info = osparc.openapi()
     assert isinstance(info["info"]["version"], str)
+
+
+def test_dependencies(tmp_path: Path):
+    """
+    Ensure packages which are imported in osparc are also specified in setup.py
+    """
+    # get imported packages
+    import_file: Path = tmp_path / "imported_packages.txt"
+    source_package: Path = _PYTHON_DIR / "client" / "osparc"
+    assert source_package.is_dir()
+    cmd: list[str] = [
+        "pipreqs",
```

Comment:
TIP: `pipreqs` and `pipdeptree` are themselves python tools. I am sure you can import some function instead of using the CLI.  In this context, I would avoid using `subprocess` . It is error prone to read back the results etc